### PR TITLE
solve the task

### PR DESCRIPTION
**Pull request by Cerne13**

### CODE REVIEW

**Review by Kitazuka — APPROVED** [review]
Looks nice

File: app/main.py [hunk 1]
```diff
@@ -1 +1,42 @@
...
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ):
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self):
+        print(f'Hello, I\'m {self.name}')
```
Comment (TPolina) [inline app/main.py:13]: use double quotes with f-string and don't use `\ '`

File: app/main.py [hunk 2]
```diff
@@ -1 +1,42 @@
...
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f'Eating {self.appetite} food points...')
+            self.is_hungry = False
+            return self.appetite
+        return 0
+
+
+class Cat(Animal):
+    def __init__(self, name, is_hungry=True):
```
Comment (TPolina) [inline app/main.py:24]: add type annotation where it is missing

**Review by TPolina — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 2 above)
Comment (Cerne13) [reply app/main.py:24]: Thank you!

**Review by TPolina — APPROVED** [review]
Well done!

**Pull request closed without merging**